src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -1827,6 +1839,10 @@ private boolean isStatsShown() {
         return showStats != null;
     }
 
+    public boolean getMissingResponsesShown() {
+        return isMissingResponsesShown;
+    }
+    
```

Comment:
`get*` returning boolean does not sound right. Even if it sounds similar to the field name, `isMissingResponseShown()` is a better name for this. Or `getIs*` even though it sounds weird.